packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -140,7 +140,6 @@ function init() {
                 'https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/docs/api.md#response-format',
             }),
           },
-          resetUpdatePayment: linkTo('routes/Subscriptions', 'subscribed'),
```

Comment:
This one is a little odd - I removed this because without it, you don't see the error at all because you're immediately redirected. Ideally it's triggered on form change, however, even if that happened, since `updateRevealed` is handled internally, I think you'd have to click on "Change" anyway to see the error was gone, so this seems OK to me.